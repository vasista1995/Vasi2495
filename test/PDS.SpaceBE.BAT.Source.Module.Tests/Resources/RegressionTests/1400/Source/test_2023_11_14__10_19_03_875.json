{
  "Device": "S7185C",
  "Module": "DSO 300-EPAD-LSS",
  "ParameterClass": "E_SPC",
  "CFComment": "EOL",
  "Segment": "MOLD",
  "F56Parameter": "N",
  "FourDReport": "*",
  "SpaceRawValues": [
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 240196694,
      "Seqnr": 4,
      "Value": 0.024,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 2,
      "ViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812",
      "ViolationList": "1,3",
      "PrimaryViolation": "1",
      "PrimaryViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812"
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 240196694,
      "Seqnr": 3,
      "Value": 0.024,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 2,
      "ViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812",
      "ViolationList": "1,3",
      "PrimaryViolation": "1",
      "PrimaryViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812"
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 240196694,
      "Seqnr": 2,
      "Value": 0.062,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 2,
      "ViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812",
      "ViolationList": "1,3",
      "PrimaryViolation": "1",
      "PrimaryViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812"
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 240196694,
      "Seqnr": 1,
      "Value": 0.045,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 2,
      "ViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812",
      "ViolationList": "1,3",
      "PrimaryViolation": "1",
      "PrimaryViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812"
    }
  ],
  "UniqueSourceKey": "BAT:BE:RU334004B03:BE1I:1700:OffSet-X:73251:240196694:61",
  "PKey": "BAT:BE:RU334004B03:BE1I:1700:OffSet-X:73251:240196694:61",
  "ParameterName": "OffSet-X",
  "ParameterUnit": "mm",
  "SampleDate": "2023-11-14T16:16:08Z",
  "SampleDateUtc": "2023-11-14T09:16:08Z",
  "SampleSize": 4,
  "Sigma": ",0183915741577495",
  "Mean": ",03875",
  "Median": ",0345",
  "Min": ",024",
  "Max": ",062",
  "Q1": ",024",
  "Q3": ",04925",
  "LdsID": "1400",
  "IsCalcParam": 0,
  "MeanCntrlTarget": 0.0162,
  "MeanCntrlHigh": 0.0377518995682515,
  "RangeCntrlTarget": 0.02388,
  "RangeCntrlHigh": 0.0603088878096163,
  "SpecTarget": 0.0,
  "SpecHigh": 0.05,
  "CtrlMeanHighEnabled": "Y",
  "CtrlMeanLowEnabled": "N",
  "CtrlMeanTargetEnabled": "Y",
  "CtrlLowEnabled": "N",
  "CtrlHighEnabled": "N",
  "CtrlRangeLowEnabled": "N",
  "CtrlRangeHighEnabled": "Y",
  "CtrlSigmaLowEnabled": "N",
  "CtrlSigmaCenterEnabled": "N",
  "CtrlRangeCenterEnabled": "Y",
  "CtrlCenterEnabled": "N",
  "CtrlSigmaHighEnabled": "N",
  "SpecLowEnabled": "N",
  "SpecHighEnabled": "Y",
  "CreatedDate": "2023-11-14T16:16:08Z",
  "CreatedDateUtc": "2023-11-14T09:16:08Z",
  "UpdatedDate": "2023-11-14T16:16:33Z",
  "UpdatedDateUtc": "2023-11-14T09:16:33Z",
  "RvStoreFlag": "Y",
  "Operation": "1700",
  "Facility": "BE1I",
  "ChannelId": "73251",
  "CkcId": "61",
  "ChannelName": "Mold_OffSet/DSO 300/X/OGP",
  "ChannelDescr": "Created from LDS \nexclude list by \naction 'Create SPC Channel'",
  "SpaceInstanceName": "BE",
  "Sitekey": "BAT",
  "CalcCreated": "11/14/2023 16:16:08",
  "ILdsID": 0,
  "AcceptFlag": "Y",
  "UpdatedBy": "iscw4ce01954pz",
  "SpecTargetOrigin": "V",
  "ViolCount": 2,
  "ExtMaMV": 0.02165625,
  "ExtEwmaMV": 0.0296378800566272,
  "ExtMsMV": 0.00494974746830583,
  "OutOfOrder": "N",
  "EwmaS": 0.0160955008457128,
  "EwmaR": 0.0351755043597517,
  "Calc2ID": 2526572,
  "LimitEnable": "NYNYNNYNYYYYYYYYYYYNNNYYYYYYNNNNNNNNN",
  "ExtEwmaMVLCL": 0.00531135609282328,
  "ExtEwmaMVCenter": 0.0162,
  "ExtEwmaMVUCL": 0.0270886439071767,
  "ExtMaMVLCL": 0.00850054781381787,
  "ExtMaMVCenter": 0.0162,
  "ExtMaMVUCL": 0.0238994521861821,
  "EwmaSLCL": 0.00348084391906827,
  "EwmaSCenter": 0.0109365879794544,
  "EwmaSUCL": 0.0183923320398405,
  "EwmaRLCL": 0.00758978076992516,
  "EwmaRCenter": 0.02388,
  "EwmaRUCL": 0.0401702192300748,
  "ExtMSMVLCL": 1.18864530321004E-05,
  "ExtMSMVCenter": 0.00560529438003064,
  "ExtMSMVUCL": 0.0225166845991629,
  "ClOrigin": 0,
  "MVLal": 0.0,
  "MVUal": 0.0,
  "RawLal": -0.0404761132945954,
  "RawUal": 0.0728761132945954,
  "SigmaLal": 0.0,
  "SigmaUal": 0.0,
  "RangeLal": 0.0,
  "RangeUal": 0.0,
  "ExtSpecLower": 0.0,
  "ExtSpecUpper": 0.0,
  "ExtSpecLimEnable": "NN",
  "CalculationStrategy": "ND",
  "ChannelType": "N",
  "ChState": "Online"
}