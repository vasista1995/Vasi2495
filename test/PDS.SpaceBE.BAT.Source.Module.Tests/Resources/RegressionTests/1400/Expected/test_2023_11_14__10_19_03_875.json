{
  "IdSource": "SPACEACT2:BAT:BE:RU334004B03:BE1I:1700:OffSet-X:73251:240196694:61",
  "Data_flat_Metadata": {
    "Device": "S7185C",
    "Module": "DSO 300-EPAD-LSS",
    "ParameterClass": "E_SPC",
    "CFComment": "EOL",
    "Segment": "MOLD",
    "F56Parameter": "N",
    "FourDReport": "*",
    "ParameterName": "OffSet-X",
    "ParameterUnit": "mm",
    "ParameterOper": "1700",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "ChannelId": "73251",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-11-14T09:16:08Z",
    "CreatedTimestamp": "2023-11-14T16:16:08Z",
    "UpdatedTimestamp": "2023-11-14T16:16:33Z",
    "UpdatedTimestampUtc": "2023-11-14T09:16:33Z",
    "ExportedTimestampUtc": "2023-11-14T16:16:08Z",
    "ExportedTimestamp": "2023-11-14T16:16:08Z",
    "SampleTimestampUtc": "2023-11-14T09:16:08Z",
    "SampleTimestamp": "2023-11-14T16:16:08Z",
    "SourceDataLevel": "C",
    "ChannelName": "Mold_OffSet/DSO 300/X/OGP",
    "ChannelDescr": "Created from LDS \nexclude list by \naction 'Create SPC Channel'",
    "RvStoreFlag": "Y",
    "CalcCreated": "11/14/2023 16:16:08",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": "iscw4ce01954pz",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 2526572,
    "ViolCount": 2,
    "CalculationStrategy": "ND",
    "LdsID": "1400",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "61",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "N",
        "SpecHigh": 0.05,
        "SpecTarget": 0.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "NYNYNNYNYYYYYYYYYYYNNNYYYYYYNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": ",03875",
        "Median": ",0345",
        "Sigma": ",0183915741577495",
        "Min": ",024",
        "Max": ",062",
        "Q1": ",024",
        "Q3": ",04925"
      },
      "ControlLimits": {
        "CtrlMeanLowEnabled": "N",
        "ctrl_mean_target": 0.0162,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 0.0377518995682515,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 0.02388,
        "ctrl_range_high": 0.0603088878096163,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 0.02165625,
        "ExtEwmaMV": 0.0296378800566272,
        "ExtMsMV": 0.00494974746830583,
        "EwmaS": 0.0160955008457128,
        "EwmaR": 0.0351755043597517,
        "ExtEwmaMVLCL": 0.00531135609282328,
        "ExtEwmaMVCenter": 0.0162,
        "ExtEwmaMVUCL": 0.0270886439071767,
        "ExtMaMVLCL": 0.00850054781381787,
        "ExtMaMVCenter": 0.0162,
        "ExtMaMVUCL": 0.0238994521861821,
        "EwmaSLCL": 0.00348084391906827,
        "EwmaSCenter": 0.0109365879794544,
        "EwmaSUCL": 0.0183923320398405,
        "EwmaRLCL": 0.00758978076992516,
        "EwmaRCenter": 0.02388,
        "EwmaRUCL": 0.0401702192300748,
        "ExtMSMVLCL": 1.18864530321004E-05,
        "ExtMSMVCenter": 0.00560529438003064,
        "ExtMSMVUCL": 0.0225166845991629,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": -0.0404761132945954,
        "RawUal": 0.0728761132945954,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 240196694,
        "Seqnr": 4,
        "Value": 0.024,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812"
      },
      {
        "SampleId": 240196694,
        "Seqnr": 3,
        "Value": 0.024,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812"
      },
      {
        "SampleId": 240196694,
        "Seqnr": 2,
        "Value": 0.062,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812"
      },
      {
        "SampleId": 240196694,
        "Seqnr": 1,
        "Value": 0.045,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "F/D \t: OFFSET X RAW ABOVE SPECIFICATION\nR/C \t: UNBALANCE DIESET\nC/A \t: REALIGMENT DIESET\nTYPE \t: ENGINEERING\nPCDF \t: NA\nTECH \t: BONDOR\nPC \t: 30045812"
      }
    ],
    "SampleSize": 4
  },
  "ProductionAction": {
    "Id": "SPACEACT1:BAT:BE:BE1I:1700:OffSet-X:73251",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "BAT:BE:RU334004B03:BE1I:1700:OffSet-X:73251:240196694:61"
  }
}